modules/33acrossBidAdapter.js

```diff
@@ -404,6 +405,16 @@ function _buildVideoORTB(bidRequest) {
   return video;
 }
 
+function _buildDeviceORTB() {
+  return {
+    ext: {
```

Comment:
the final version of this function would look like this:
```
 const win = utils.getWindowSelf();

  return {
    ...getScreenDimensions(),
    pxratio: win.devicePixelRatio,
    ext: {
      ttx: {
        viewport: getViewportDimensions(),
        availheight: win.screen.availHeight,
        maxtouchpoints: win.navigator.maxTouchPoints
      }
    }
  }
```

following your refactoring suggestion, it'd look like this:
```
 const win = utils.getWindowSelf();
 const device = {
    ...getScreenDimensions(),
    pxratio: win.devicePixelRatio,
  }
  device = setExtension(device, 'viewport',  getViewportDimensions());
  device = setExtension(device, 'availheight',  win.screen.availHeight);
  device = setExtension(device, 'maxtouchpoints',  win.navigator.maxTouchPoints);
  return device;
```

I am not sure about this new version because it isn't that DRY. so, wondering if I could refactor `setExtension` so it accepts a group of key-value pairs. 
e.g.
```
function setExtensions(obj = {}, extFields) {
  return Object.assign({}, obj, {
    ext: Object.assign({}, obj.ext, extFields)
  });
}
```

Reply:
Ah got it, so the reason I created `setExtensions` util was that when you have multiple collaborators working on the adapter at different points of time, they started sticking code overriding extensions all over. My hope was to have this util safeguard against that. 

Yes, please feel free to refactor it if you think it'll be more useful that way.